pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[build-system]
+requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=3.4"]
+build-backend = "setuptools.build_meta"
+
+# Enable setuptools_scm to compute the version number from the most recent tag
+# https://github.com/pypa/setuptools_scm/#pyprojecttoml-usage
```

Comment:
Is there a reason for disabling `setuptools_scm`?

Reply:
not at all, but had to test how it works. `setuptools_scm` collects the version from git which is not supported in pypi therefore it was added the test version in setup  manual

Reply:
in theory I should create a tag with the correct version and uncomment it

Reply:
fixed